Paralayout/UIView+Alignment.swift

```diff
@@ -115,6 +115,80 @@ extension Alignable {
         return sourcePoint.offset(to: targetPoint)
     }
 
+    /// Calculates the offset between two views' positions, ignoring any transforms in the view hierarchy.
+    ///
+    /// - precondition: The receiver and `otherView` must be in the same view hierarchy.
+    ///
+    /// - parameter position: The position in the receiving view's untransformed frame.
+    /// - parameter otherView: The other view for the measurement.
+    /// - parameter otherPosition: The position in the `otherView`'s untransformed frame to use for the measurement.
+    /// - returns: The offset from the receiver's `position` to the `otherView`'s `otherPosition`.
+    public func untransformedFrameOrigin(
```

Comment:
A couple things to consider:
1. We should probably eliminate the `untransformedFrameOffset` method entirely -- but that's a breaking change
2. If not, we ought to DRY up the implementations, of course

Reply:
1. I'm okay with breaking changes. We've been trending towards a breaking release for a bit now. `main` already has breaking changes on it – [example](https://github.com/square/Paralayout/pull/140#discussion_r1821969282). One big thing though is the documentation for this method does not make it clear how it differs from `untransformedFrameOffset`, so I'm not exactly sure from reading the doc/name how I'd migrate from one to the other. (Haven't read the implementation just yet)
2. heh, yeah. We've been finding that difficult... been resorting to copy/paste [recently](https://github.com/square/Paralayout/pull/140)

Reply:
yeah I'm struggling with this function name + parameter names